mods/ra/maps/soviet-05/main.lua

```diff
@@ -0,0 +1,276 @@
+						if #fact == 0 then
+							return
+						else
+							Trigger.OnDamaged(fact[1], function()
+								if fact[1].Owner == GoodGuy and fact[1].Health < fact[1].MaxHealth * 3/4 then
+									fact[1].StartBuildingRepairs()
+								end
+							end)
+						end
+					end)
+
+					IslandTroops1()
+					Trigger.AfterDelay(DateTime.Minutes(3), IslandTroops2)
+					Trigger.AfterDelay(DateTime.Minutes(6), IslandTroops3)
+					Trigger.AfterDelay(DateTime.Seconds(7), BuildBase)
+				end
+
+				if not mcvtransport.IsDead then
+					mcvtransport.Move(ReinfNorthPoint.Location)
+					mcvtransport.Destroy()
+				end
+			end)
+		end)
+	end)
+end
+
+Tick = function()
+	if Greece.HasNoRequiredUnits() and GoodGuy.HasNoRequiredUnits() then
+		player.MarkCompletedObjective(KillAll)
+		player.MarkCompletedObjective(HoldObjective)
+	end
+
+	if player.HasNoRequiredUnits() then
+		GoodGuy.MarkCompletedObjective(BeatUSSR)
+	end
+
+if Greece.Resources >= Greece.ResourceCapacity * 0.75 then
+	Greece.Cash = Greece.Cash + Greece.Resources - Greece.ResourceCapacity * 0.25
+	Greece.Resources = Greece.ResourceCapacity * 0.25
+end
+ 
+if GoodGuy.Resources >= GoodGuy.ResourceCapacity * 0.75 then
+	GoodGuy.Cash = GoodGuy.Cash + GoodGuy.Resources - GoodGuy.ResourceCapacity * 0.25
+	GoodGuy.Resources = GoodGuy.ResourceCapacity * 0.25
+end
+
+	if not baseEstablished and CheckForBase() then
+		baseEstablished = true
+		Para()
+	end
+
+	if not SPenEstablished and CheckForSPen() then
+		SPenEstablished = true
+
+		local units = Reinforcements.ReinforceWithTransport(Greece, "lst", ArtyReinf, SouthReinfPath, { ReinfEastPoint.Location })[2]
+		Utils.Do(units, function(unit) IdleHunt(unit) end)
+		if not ExpansionCheck then
+			Expand()
+			ExpansionCheck = true
+		end
+	end
+
+	if not RCheck then
+		RCheck = true
+		if Map.Difficulty == "Easy" and ReinfCheck then
+			Trigger.AfterDelay(DateTime.Minutes(6), ReinfArmor)
+		elseif Map.Difficulty == "Medium" then
+			Trigger.AfterDelay(DateTime.Minutes(4), ReinfArmor)
+		else
+			Trigger.AfterDelay(DateTime.Minutes(3), ReinfArmor)
+		end
+	end
+end
+
+WorldLoaded = function()
+	player = Player.GetPlayer("USSR")
+	GoodGuy = Player.GetPlayer("GoodGuy")
+	Greece = Player.GetPlayer("Greece")
+
+	RunInitialActivities()
+
+	Trigger.OnObjectiveAdded(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "New " .. string.lower(p.GetObjectiveType(id)) .. " objective")
+	end)
+	Trigger.OnObjectiveCompleted(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective completed")
+		Media.PlaySpeechNotification(player, "ObjectiveMet")
+	end)
+	Trigger.OnObjectiveFailed(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective failed")
+	end)
+
+	CaptureObjective = player.AddPrimaryObjective("Capture the Radar Dome.")
+	KillAll = player.AddPrimaryObjective("Defeat the Allied forces.")
+	BeatUSSR = GoodGuy.AddPrimaryObjective("Defeat the USSR forces.")
```

Comment:
this string is probably not displayed ingame, but the USSR forces are called "Soviet forces" in all the other missions